slave.mk

```diff
@@ -398,6 +398,7 @@ $(info "ENABLE_AUTO_TECH_SUPPORT"        : "$(ENABLE_AUTO_TECH_SUPPORT)")
 $(info "PDDF_SUPPORT"                    : "$(PDDF_SUPPORT)")
 $(info "MULTIARCH_QEMU_ENVIRON"          : "$(MULTIARCH_QEMU_ENVIRON)")
 $(info "SONIC_VERSION_CONTROL_COMPONENTS": "$(SONIC_VERSION_CONTROL_COMPONENTS)")
+$(info "SONIC_BUILD_RETRY"               : "$(SONIC_BUILD_RETRY)")
```

Comment:
How about change the SONIC_BUILD_RETRY to SONIC_BUILD_RETRY_TIME?

If SONIC_BUILD_RETRY_TIME not defined, or 0, not retry
If SONIC_BUILD_RETRY_TIME > 0, then retry it.
If SONIC_BUILD_RETRY_TIME any other not valid values, not retry.

Support to set the waiting interval SONIC_BUILD_RETRY_INTERNVAL, wait time in each of the retry in seconds.